code/modules/client/client_defines.dm

```diff
@@ -114,6 +114,9 @@
 	/// Is the client watchlisted
 	var/watchlisted = FALSE
 
+	/// Client's pAI save
+	var/datum/pai_save/pai_save
```

Comment:
Someone (who can do this in a PR that isnt mine) can make this in a PR


Add a tab to settings to bring up the user's own pAI save and let them edit it there, since these datums now exist always.